<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXEditPathFile</class>
 <widget class="QWidget" name="CXEditPathFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="QGroupBox" name="mTitleGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="margin">
       <number>7</number>
      </property>
      <item>
       <widget class="QTextEdit" name="mTextEdit">
        <property name="textInteractionFlags">
         <set>Qt::TextEditorInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mCurrentLineLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="CXTouchButton" name="mSaveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>139</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="CXTouchButton" name="mSaveAsButton">
     <property name="text">
      <string>Сохранить как</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="CXTouchButton" name="mCreateButton">
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="CXTouchButton" name="mStatButton">
     <property name="text">
      <string>Стат</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>cxtouchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
